include/LLGL/Platform/Linux/LinuxNativeHandle.h

```diff
@@ -36,6 +39,63 @@ struct NativeHandle
     int             screen;
 };
 
+#if LLGL_LINUX_ENABLE_WAYLAND
+
+struct WaylandNativeHandle
+{
+    struct wl_surface* window;
+    struct wl_display* display;
+};
+
+#else
+
+struct WaylandNativeHandle
+{
+    void* window;
+    void* display;
+};
+
+#endif
+
+enum class NativeHandleType : char
```

Comment:
Just as mentioned above with `RenderSystemNativeHandleType`, if we name it `RenderSystemNativeType` we should also rename this one to `NativeType`. I wanted to rename the "NativeHandle" to "SurfaceNativeHandle" at some point to clearly differentiate it from the render system native handles, but we might also just keep it as-is.